web/src/lib/domain-logic/roadmapWithSectionSpecificTasks.ts

```diff
@@ -0,0 +1,22 @@
+import { Roadmap, Step, Task } from "@/lib/types/types";
+import { SectionType } from "@businessnjgovnavigator/shared/userData";
+import { cloneDeep } from "lodash";
+
+export const roadmapWithSectionSpecificTasks = (roadmap: Roadmap, section: SectionType): Roadmap => {
+  const result = { steps: [], tasks: [] } as { steps: Step[]; tasks: Task[] };
```

Comment:
Again, we should avoid the `as`.

Can we do `const result: Roadmap = { steps: [], tasks: [] }` ?